code/datums/wires/wires.dm

```diff
@@ -71,15 +71,15 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 		src.wires[colour] = index
 		//wires = shuffle(wires)
 
-/datum/wires/proc/examine(index, mob/user)
+/datum/wires/proc/examine(index, mob/user,)
 	. = "You aren't sure what this wire does."
+	var/mec_stat = user.stats.getStat(STAT_MEC)
 
 	var/datum/wire_description/wd = get_description(index)
 	if(!wd)
 		return
-	//TODO: Port bay wires fully and integrate eris' skill system
-	//if(wd.skill_level && !user.skill_check(SKILL_ELECTRICAL, wd.skill_level))
-		//return
+	if(wd.skill_level > mec_stat)
+		return
```

Comment:
Make it percentage based, as in 20/40 skill is 50% chance to get the right wire description and 50% chance to get a random wire description instead. That would be funny, I think.

Reply:
Nah, then you'd be able to just figure out all the wires eventually, or never be sure the examine is correct
Anyway, bad idea, github mobile is lagging too muvh for me to make a better argument